tests/api/test_pagination.py

```diff
@@ -0,0 +1,1798 @@
+    MenuItem.objects.create(name="MenuItem6", menu=menu3)
+    MenuItem.objects.create(name="MenuItem7", menu=menu3)
+    MenuItem.objects.create(name="MenuItem8", menu=menu3)
+    MenuItem.objects.create(name="MenuItem9", menu=menu3)
+
+
+@pytest.fixture
+def orders_for_pagination(customer_user):
+    address1 = Address.objects.create(
+        first_name="Lily",
+        last_name="Allen",
+        company_name="Mirumee Software",
+        street_address_1="Tęczowa 7",
+        city="WROCŁAW",
+        postal_code="53-601",
+        country="PL",
+        phone="+48713988102",
+    )
+    address2 = Address.objects.create(
+        first_name="Bob",
+        last_name="Dylan",
+        company_name="Mirumee Software",
+        street_address_1="Tęczowa 7",
+        city="WROCŁAW",
+        postal_code="53-601",
+        country="PL",
+        phone="+48713988102",
+    )
+    address3 = Address.objects.create(
+        first_name="Clark",
+        last_name="Kent",
+        company_name="Mirumee Software",
+        street_address_1="Tęczowa 7",
+        city="WROCŁAW",
+        postal_code="53-601",
+        country="PL",
+        phone="+48713988102",
+    )
+    address4 = Address.objects.create(
+        first_name="Danny",
+        last_name="Devito",
+        company_name="Mirumee Software",
+        street_address_1="Tęczowa 7",
+        city="WROCŁAW",
+        postal_code="53-601",
+        country="PL",
+        phone="+48713988102",
+    )
+    address5 = Address.objects.create(
+        first_name="Ellen",
+        last_name="Degeneres",
+        company_name="Mirumee Software",
+        street_address_1="Tęczowa 7",
+        city="WROCŁAW",
+        postal_code="53-601",
+        country="PL",
+        phone="+48713988102",
+    )
+
+    order1 = Order.objects.create(
+        user=customer_user,
+        status=OrderStatus.FULFILLED,
+        token="order_token_1",
+        total_gross_amount=Decimal("10"),
+        billing_address=address1,
+    )
+    Order.objects.create(
+        user=customer_user,
+        status=OrderStatus.FULFILLED,
+        token="order_token_2",
+        total_gross_amount=Decimal("20"),
+        billing_address=address2,
+    )
+    order3 = Order.objects.create(
+        user=customer_user,
+        status=OrderStatus.FULFILLED,
+        token="order_token_3",
+        total_gross_amount=Decimal("10"),
+        billing_address=address3,
+    )
+    order4 = Order.objects.create(
+        user=customer_user,
+        status=OrderStatus.FULFILLED,
+        token="order_token_4",
+        total_gross_amount=Decimal("20"),
+        billing_address=address4,
+    )
+    order5 = Order.objects.create(
+        user=customer_user,
+        status=OrderStatus.FULFILLED,
+        token="order_token_5",
+        total_gross_amount=Decimal("50"),
+        billing_address=address5,
+    )
+
+    Payment.objects.bulk_create(
+        [
+            Payment(
+                gateway="Dummy",
```

Comment:
What about some pytest factories to cover these objects? The same thing for order and address. Here you have a lot of duplicated code.